js_modules/dagster-ui/packages/ui-core/src/workspace/VirtualizedJobRow.tsx

```diff
@@ -119,7 +117,11 @@ export const VirtualizedJobRow = (props: JobRowProps) => {
         </RowCell>
         <RowCell>
           <Box flex={{justifyContent: 'flex-end'}} style={{marginTop: '-2px'}}>
-            <JobMenu job={{isJob, name, runs: latestRuns}} repoAddress={repoAddress} />
+            <JobMenu
+              job={{name, isJob, runs: latestRuns}}
+              isAssetJob={pipeline ? pipeline.isAssetJob : 'loading'}
```

Comment:
Sending this separately from `job` because I wanted to add an explicit loading state, so that until the `SINGLE_JOB_QUERY` completes it says "Loading" instead of the action potentially assuming isAssetJob=false